### PR TITLE
Implement Discount section

### PR DESCRIPTION
**Pull request by JennyDoJD**
**Design**
Mobile
image
Desktop
image

**Result**
Mobile
image
Tablet
image
Desktop
image

### CODE REVIEW

File: practice/src/css/sections/discount.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
+.wrapper-discount {
```
Comment (huyencao-agilityio) [inline practice/src/css/sections/discount.css:1]: .wrapper-discount {
  position: relative;
}
Douple check

File: practice/src/index.html [hunk 2]
```diff
@@ -70,6 +70,42 @@ <h5 class="service-topic-title">Easy trip booking</h5>
...
+
+    <section class="discount-section">
+      <div class="wrapper-discount">
+        <div class="container discount-main">
+          <img src="../src/assets/images/icon-heart.svg" alt="" class="icon-heart">
+          <div class="discount-body">
+            <hgroup class="text discount-heading">
+                <h2 class="main-title ">For beloved customers</h2>
+                <h3 class="secondary-title discount-title">20% discount if you have used our trip services!</h3>
+                <p class="discount-description description">Let's maximize your next holiday with us with the best experience.</p>
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Should move common class to first

File: practice/src/index.html [hunk 3]
```diff
@@ -70,6 +70,42 @@ <h5 class="service-topic-title">Easy trip booking</h5>
...
+            </hgroup>
+            <div class="discount-btn">
+              <a href="javascrip:void(0)" class="btn btn-primary btn-claim">Claim Now</a>
+            </div>
+          </div>
+            <div class="text discount-comment">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Code style

File: practice/src/index.html [hunk 4]
```diff
@@ -70,6 +70,42 @@ <h5 class="service-topic-title">Easy trip booking</h5>
...
+                <img src="../src/assets/images/icon-star.svg" class="icon-star">
+                <img src="../src/assets/images/icon-star.svg" class="icon-star">
+                <img src="../src/assets/images/icon-star.svg" class="icon-star">
+                <img src="../src/assets/images/icon-star.svg" class="icon-star">
+                <img src="../src/assets/images/icon-star.svg" class="icon-star">
+              </div>
+              <p class="comment-customer">Thank you for helping my vacation. Amazing!</p>
+              <div class="information-customer">
+                <img src="../src/assets/images/discount/avatar-rojali.png" class="avatar-rojali">
+                <hgroup class="rojali-info">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Incorrect tag and update class name

File: practice/src/css/sections/discount.css [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+    align-items: center;
+  }
+
+  .icon-heart {
+    transform: translate(-10%, -50%);
+  }
+
+  .discount-comment {
+    display: block;
+    position: absolute;
```
Comment (huyencao-agilityio) [inline practice/src/css/sections/discount.css:89]: why do you need to use position absolute here?

File: practice/src/css/sections/discount.css [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+  padding-top: 40px;
+}
+
+.discount-heading {
+  text-align: center;
+  padding-bottom: 99px;
+}
+
+.btn-claim {
+  font-size: 14px;
```
Comment (huyencao-agilityio) [inline practice/src/css/sections/discount.css]: Use variable

File: practice/src/css/sections/discount.css (hunk 5 above)
Comment (JennyDoJD) [reply practice/src/css/sections/discount.css:89]: I want to set its position based on the parent element which is .wrapper-discount with position: relative

**Comment by huyencao-agilityio** [comment]
@JennyDoJD Conflict

**Pull request merged**